products/sce/container-groups/imds/instance-deletion-cost.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Instance Deletion Cost'
+description: 'Manage scale-in behavior for container groups by setting the instance deletion cost.'
+---
+
+## Overview
+
+When a salad container group is scaled in via adjusting its replica count, the instance deletion cost determines how the
+system selects which instances to terminate. The instance deletion cost is a numeric value that you can assign to each
+instance in the container group. Instances with a lower deletion cost are prioritized for termination when scaling down.
+
+Assigning this value can be done through the instance metadata service, using the
+[Replace Deletion Cost Endpoint](/reference/imds/replace-deletion-cost). This function is also available in
+SaladCloudIMDS SDKs.
+
+The deletion cost is an integer value, and the default is `0`. You can set it to any integer value, including negative
+numbers. This allows you to prioritize less busy instances for termination by assigning them a lower deletion cost,
+while keeping more critical instances with a higher deletion cost.
+
+## Example 1: Batch Model Finetuning
+
+Suppose you have a container group running dozens or hundreds of instances, each pulling training jobs from a queue. In
+this situation, you would want idle nodes to set their deletion cost to 0, while nodes that are actively doing work
+could gradually increase their deletion cost as the job progresses. This way, if replica count is reduced, the system
+would not interrupt the training job by terminating an instance that is currently processing a task, but rather
+terminate an idle instance instead.
+
+## Example 2: Image Generation with ComfyUI
+
+In a scenario where you are using ComfyUI for large-scale image generation, instances may accumulate work in their
+internal queues. You can set the deletion cost of the instance to the depth of the queue, so that fewer jobs are lost
+when scaling down. For example, if an instance has a queue depth of 5, you can set its deletion cost to 5. This way, if
+the system needs to scale down, it will prioritize terminating instances with a lower deletion cost, thus preserving
+instances that are actively processing jobs in their queues.
+
+## Conclusion
+
+Setting the instance deletion cost is a powerful and flexible way to manage scale-in behavior for container groups.
+Note, however, that this deletion cost only applies when intentional scaling events occur, such as setting replica
```

Comment:
This is an important note, and K8s explicitly highlights it in their docs. Should we use a callout box and/or note it before the "conclusion"?